### ISSUE TITLE
Styling is broken for modals.

### ISSUE THREAD

**Issue by diclee** [koding/koding#10471]
## Current Behavior
![image](https://cloud.githubusercontent.com/assets/8261892/22534275/d3b0e22a-e8a7-11e6-8caf-7bc5a09981a2.png)

![gif](http://g.recordit.co/XoYpsFjuLV.gif)
http://recordit.co/XoYpsFjuLV

Order of style files might be causing the problem.

**Comment by gokmen** [comment]
I believe this is happening because of a change in order of styles, kd.js styles looks like moved to top in import order. @usirin can this be related with your code splitting changes?

**Comment by usirin** [comment]
@gokmen You're right. now we are shipping 2 different css files and kd.js is on the `bundle.vendor.css` and it's being included before the `bundle.main.css`. It should already be like this already don't you think?

**Comment by diclee** [comment]
@usirin minor thing remained that also needs to be fixed:
![image](https://cloud.githubusercontent.com/assets/8261892/22607994/58d219fc-ea0f-11e6-8686-13cdb35e1424.png)

**Comment by cansirin** [comment]
It's updated. Can you check it again please?

**Comment by diclee** [comment]
@cansirin only fixed for credential preview modals.

still reproducible for:
- [ ] remove credential
- [ ] readme preview
- [ ] stack template preview
- [ ] delete stack template modal
- [ ] add your own machine modal
- [ ] edit credential
- [ ] verify current password modals
- [ ] admin invitation warning modal
- [ ] resend invitation
- [ ] invites already sent modal

**Comment by diclee** [comment]
Also the styling for buttons on 'Edit Credential' modal is broken:
![image](https://cloud.githubusercontent.com/assets/8261892/22808635/e55edb7c-eee1-11e6-8e1c-4a4d950f79c6.png)

**Comment by usirin** [comment]
@cansirin button spacing in `Edit Credential` modal also needs to be fixed.

**Comment by diclee** [comment]
Verify passed on sandbox for the original issue.
